includes/class-object-cache.php

```
@@ -14,299 +14,307 @@
-		$config_data = $config;
+			// Write config file using var_export for clean array representation.
+			$config_content = "<?php\n/**\n * Auto-generated by Performance Optimisation\n */\n\nif ( ! defined( 'ABSPATH' ) ) {\n\texit;\n}\n\n";
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_var_export
+			$config_content .= 'return ' . var_export( $config_data, true ) . ";\n";
 
-		// Remove password from config file — sensitive credentials are stored
-		// in the `wppo_settings` option (database) and merged at load time via
-		// the `WPPO_REDIS_PASSWORD` constant, environment variable, or the
-		// object cache's REST handler.
-		unset( $config_data['password'] );
+			$wp_filesystem = Util::init_filesystem();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Util' in method 'enable'. (undefined)

(StaticAccess)